test/specs/lib.davIdIcon.spec.js

```diff
@@ -17,4 +17,18 @@ describe('getDavIdIconUrl()', () => {
       return false;  
     }
   };
+
+  test('Return contains the davID within it options',() => {     
```

Comment:
Please change the description of the test to:
`returns a string containing the given davID`